### DISCUSSION TITLE
error in ww3_shel

### DISCUSSION THREAD

**Discussion by ppokhrel1** [NOAA-EMC/WW3#570] [Q&A]
I am gettting the following error with Katrina T03 Tutorial.

                     *** WAVEWATCH III Program shell ***      
               ===============================================

  Comment character is '$'

At line 923 of file ww3_shel.F90 (unit = 10, file = './ww3_shel.inp')
Fortran runtime error: Bad repeat count in item 2 of list input

Error termination. Backtrace:
#0  0x7f16e9d1d32a
#1  0x7f16e9d1ded5
#2  0x7f16e9d1e69d
#3  0x7f16e9e8b005
#4  0x7f16e9e8b098
#5  0x7f16e9e8dd02
#6  0x7f16e9e8f379
#7  0x555fe8f2a5ed
#8  0x555fe8f29e08
#9  0x7f16e937cbf6
#10  0x555fe8f29e39
#11  0xffffffffffffffff

I am using 6.07.1 version. Could anyone please help? 

Shel input file:
$ -------------------------------------------------------------------- $
$ WAVEWATCH III shell input file                                       $
$ -------------------------------------------------------------------- $
$ Define input to be used with flag for use and flag for definition
$ as a homogeneous field; four input lines.
$ 1) Water levels
$ 2) Currents
$ 3) Winds
$ 4) Ice concentrations (cannot be homogeneous).
$ 5,6,7)  Assimilation data
F F
F F
T F
F
F
F
F
$
$ Time frame of calculations ----------------------------------------- $
$ - Starting time in yyyymmdd hhmmss format.
$ - Ending time in yyyymmdd hhmmss format.
$
20050828 120000
20050830 000000
$
   1
$
$ Define output data ------------------------------------------------- $
$ Five output types are available (see below). All output types share
$ a similar format for the first input line:
$ - first time in yyyymmdd hhmmss format, output interval (s), and
$   last time in yyyymmdd hhmmss format (all integers).
$ Output is disabled by setting the output interval to 0.
$
$          Fields of mean wave parameters
$          Standard line and line with flags to activate output fields
$          as defined in section 2.4 of the manual. The second line is
$          not supplied if no output is requested.
$                               The raw data file is out_grd.ww3,
$                               see w3iogo.ftn for additional doc.
$
   20050828 120000  3600  20050830 000000
$
  N
  HS FP DP
$
$         Point output
$          Standard line and a number of lines identifying the
$          longitude, latitude and name (C*10) of output points.
$          The list is closed by defining a point with the name
$          'STOPSTRING'. No point info read if no point output is
$          requested (i.e., no 'STOPSTRING' needed).
$                               The raw data file is out_pnt.ww3,
$                               see w3iogo.ftn for additional doc.
$
   20050828 120000  3600  20050830 000000
$
  N
  HS FP DP
$
$         Point output
$          Standard line and a number of lines identifying the
$          longitude, latitude and name (C*10) of output points.
$          The list is closed by defining a point with the name
$          'STOPSTRING'. No point info read if no point output is
$          requested (i.e., no 'STOPSTRING' needed).
$                               The raw data file is out_pnt.ww3,
$                               see w3iogo.ftn for additional doc.
$
   20050828 120000  3600  20050830 000000
$
    271.774 29.208 'NDBC42040'
     0.0   0.0  'STOPSTRING'
$
$ Output along  track.
   20050828 120000     0  20050830 000000
$ Restart files (no additional data required). The data file is restartN.ww3, see w3iors.ftn for additional doc.
   20050830 000000     0  20050830 000000
$ Boundary data (no additional data required). The data file is nestN.ww3, see w3iobp.ftn for additional doc.
   20050828 120000     0  20050830 000000
$ Separated wave field data (dummy for now)
   20050828 120000     0  20050830 000000
$
   'STP'
$ -------------------------------------------------------------------- $
$ End of input file                                                    $

**Comment by JessicaMeixner-NOAA** [comment]
Have you tried adding more debug statements to the compiler so that you might get a more helpful debug message?  See: https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#how-to-debug-error-messages for more information.

**Comment by ppokhrel1** [comment]
Thanks Jessica,


The error I get is this for ww3_shel:

  Comment character is '$'

At line 923 of file ww3_shel.F90 (unit = 10, file = './ww3_shel.inp')
Fortran runtime error: Bad repeat count in item 2 of list input

Error termination. Backtrace:
#0  0x14865aac132a
#1  0x14865aac1ed5
#2  0x14865aac269d
#3  0x14865ac2f005
#4  0x14865ac2f098
#5  0x14865ac31d02
#6  0x14865ac33379
#7  0x55f6080835b2
#8  0x55f608082e08
#9  0x14865a120bf6
#10  0x55f608082e39
#11  0xffffffffffffffff

Any help would be greatly appreciated.


Contents of ww3_shel.inp:

$ -------------------------------------------------------------------- $
$ WAVEWATCH III shell input file                                       $
$ -------------------------------------------------------------------- $
$ Define input to be used with flag for use and flag for definition
$ as a homogeneous field; four input lines.
$ 1) Water levels
$ 2) Currents
$ 3) Winds
$ 4) Ice concentrations (cannot be homogeneous).
$ 5,6,7)  Assimilation data
   F F
   F F
   T F
   F
   F
   F
   F
$
$ Time frame of calculations ----------------------------------------- $
$ - Starting time in yyyymmdd hhmmss format.
$ - Ending time in yyyymmdd hhmmss format.
$
   20050828 120000
   20050830 000000
$
   1
$
$ Define output data ------------------------------------------------- $
$ Five output types are available (see below). All output types share
$ a similar format for the first input line:
$ - first time in yyyymmdd hhmmss format, output interval (s), and
$   last time in yyyymmdd hhmmss format (all integers).
$ Output is disabled by setting the output interval to 0.
$
$          Fields of mean wave parameters
$          Standard line and line with flags to activate output fields
$          as defined in section 2.4 of the manual. The second line is
$          not supplied if no output is requested.
$                               The raw data file is out_grd.ww3,
$                               see w3iogo.ftn for additional doc.
$
   20050828 120000  3600  20050830 000000
$
  N
  HS FP DP
$
$         Point output
$          Standard line and a number of lines identifying the
$          longitude, latitude and name (C*10) of output points.
$          The list is closed by defining a point with the name
$          'STOPSTRING'. No point info read if no point output is
$          requested (i.e., no 'STOPSTRING' needed).
$                               The raw data file is out_pnt.ww3,
$                               see w3iogo.ftn for additional doc.
$
   20050828 120000  3600  20050830 000000
$
    271.774 29.208 'NDBC42040'
     0.0   0.0  'STOPSTRING'
$
$ Output along  track.
   20050828 120000     0  20050830 000000
$ Restart files (no additional data required). The data file is restartN.ww3, see w3iors.ftn for additional doc.
   20050830 000000     0  20050830 000000
$ Boundary data (no additional data required). The data file is nestN.ww3, see w3iobp.ftn for additional doc.
   20050828 120000     0  20050830 000000
$ Separated wave field data (dummy for now)
   20050828 120000     0  20050830 000000
$
   'STP'
$ -------------------------------------------------------------------- $
$ End of input file                                                    $
$ —————————————————————————————————— $


Thanks,
Pujan Pokhrel

On Jan 6, 2022, at 7:41 AM, Jessica Meixner ***@***.******@***.***>> wrote:


WARNING: This email originated outside of the University of New Orleans system. The sender of this email could not be validated and may not actually be the person in the “From” field. Do NOT click links or open attachments if the message seems suspicious in any way. Never provide your user ID or password.




Have you tried adding more debug statements to the compiler so that you might get a more helpful debug message? See: https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#how-to-debug-error-messages<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNOAA-EMC%2FWW3%2Fwiki%2FFAQs-page%23how-to-debug-error-messages&data=04%7C01%7Cppokhrel%40uno.edu%7C3683202b0c184256bdbf08d9d11a44bd%7C31d4dbf540044469bfeedf294a9de150%7C0%7C0%7C637770733040290757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=GmdR0rCvRSWPZHXQl90SMzio073trz3CcOY8BDKbbnM%3D&reserved=0> for more information.

—
Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNOAA-EMC%2FWW3%2Fdiscussions%2F570%23discussioncomment-1918570&data=04%7C01%7Cppokhrel%40uno.edu%7C3683202b0c184256bdbf08d9d11a44bd%7C31d4dbf540044469bfeedf294a9de150%7C0%7C0%7C637770733040290757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=nnswZwb92kPn2rsw1Cf05vdoNFbYH4Cso5oVK%2BOZBGA%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACDN4B54UDMQINF2WZD3BTTUUWLZHANCNFSM5LL4K4DA&data=04%7C01%7Cppokhrel%40uno.edu%7C3683202b0c184256bdbf08d9d11a44bd%7C31d4dbf540044469bfeedf294a9de150%7C0%7C0%7C637770733040290757%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=FeBLenLwvCj5Zm51TUVmWyMrYchcpUvN838%2F5Z5y2CQ%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cppokhrel%40uno.edu%7C3683202b0c184256bdbf08d9d11a44bd%7C31d4dbf540044469bfeedf294a9de150%7C0%7C0%7C637770733040446992%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=yHN66Pt0b5lEeu%2FmHxvwfrgfHvAkLeTkOkiynMuyQoI%3D&reserved=0> or Android<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cppokhrel%40uno.edu%7C3683202b0c184256bdbf08d9d11a44bd%7C31d4dbf540044469bfeedf294a9de150%7C0%7C0%7C637770733040446992%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=91Y9zdtIadry3%2FmaDSVtO2ILvcbSkep6U3Yf5Uwd94Q%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>


NOTICE: This message, including all attachments transmitted with it, is intended solely for the use of the Addressee(s) and may contain information that is PRIVILEGED, CONFIDENTIAL, and/or EXEMPT FROM DISCLOSURE under applicable law. If you are not the intended recipient, you are hereby notified that any disclosure, copying, distribution, or use of the information contained herein is STRICTLY PROHIBITED. If you received this communication in error, please destroy all copies of the message, whether in electronic or hard copy format, as well as attachments and immediately contact the sender by replying to this email or contact the sender at the telephone numbers listed above. Thank you!

**Comment by JessicaMeixner-NOAA** [comment]
I've seen this error be a few things.  Whether it's something in the input file itself or the job submission was not set-up correctly.  Again, I'll point you to the FAQ page https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#how-to-debug-error-messages for more information.
